classes/class.ilObjInteractiveVideoGUI.php

```diff
@@ -2392,7 +2409,6 @@ public function postVideoStartedPerAjax()
 	{
 		global $ilUser;
 		$this->object->saveVideoStarted($this->obj_id, $ilUser->getId());
-		$this->object->trackProgress();
```

Comment:
I has to remove this line to have a correct LP behaviour.

Reply:
@chfsx What was the issue with this call?

P.S.: Thanks a lot for this PR!

Reply:
LP did not work correct with this line, I wasn't able to complete an IV-Item